bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -28,77 +22,166 @@ spec:
         openshift.io/component: network
         kubernetes.io/os: "linux"
     spec:
-      # Requires fairly broad permissions - ability to read all services and network functions as well
-      # as all pods.
       serviceAccountName: ovn-kubernetes-node
       hostNetwork: true
       hostPID: true
       priorityClassName: "system-node-critical"
+      # volumes in all containers:
+      # (container) -> (host)
+      # /etc/openvswitch -> /var/lib/openvswitch - ovsdb schema (selinux blocks /etc on host)
+      # /var/lib/openvswitch -> /var/lib/openvswitch - ovsdb data
+      # /run/openvswitch -> /run/openvswitch - ovsdb sockets
+      # configmap ovn-env-overrides -> /env - debug overrides
       containers:
-      # ovsdb-server and ovs-switchd daemons
       - name: ovs-daemons
         image: {{.OvnImage}}
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -euo pipefail
+          if [[ -f "/env/${K8S_NODE}" ]]; then
+            set -o allexport
+            source "/env/${K8S_NODE}"
+            set +o allexport
+          fi
+          chown -R openvswitch:openvswitch /run/openvswitch
+          chown -R openvswitch:openvswitch /etc/openvswitch
+          function quit {
+              /usr/share/openvswitch/scripts/ovs-ctl stop
+              exit 0
+          }
+          trap quit SIGTERM
+          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --no-ovs-vswitchd --system-id=random
 
-        command: ["/root/ovnkube.sh", "ovs-server"]
+          if [[ `nproc` -gt 12 ]]; then
```

Comment:
This is unnecessary with openvswitch greater than mumble mumble and we should eventually remove it

Reply:
as in, in this PR?

Reply:
if we already depend on ovs > mumble then yes, otherwise we should leave a comment with the correct value of "mumble" in it

which, ftr, is "`openvswitch2.11-2.11.0-11.el7fdn`"

Reply:
Trying to figure out which version we have. None of the `--version` commands seem to give me a commit hash.

Reply:
Aha, looks like we have openvswitch2.11.x86_64 0:2.11.0-18.el7fdp. That particular bit of cargo-culting can die.

Reply:
Wait, I'm not familiar with the suffixes. Is there any chance that `el7fdp` is different from `el7fdn`?

Reply:
> Wait, I'm not familiar with the suffixes. Is there any chance that el7fdp is different from el7fdn?

um... I think one is like testing and the other is release or somethign like that... @knobunc should know

Reply:
@squeed @danwinship ovn/ovs is from fast datapath.

Reply:
@squeed 
"fdn" = Fast Datapath Next (eg, stuff still under testing and not yet released to customers)
"fdp" = Fast Data Path (eg stuff that has been officially QA-ed and released)

Reply:
so it sounds like we *can* get rid of this

Reply:
done.

Reply:
gotten rid of.